rest_framework/throttling.py

```diff
@@ -136,7 +140,7 @@ def throttle_success(self):
         Inserts the current request's timestamp along with the key
         into the cache.
         """
-        self.history.insert(0, self.now)
+        self.history.appendleft(self.now)
         self.cache.set(self.key, self.history, self.duration)
```

Comment:
Some cache backends for Django (e.g. Redis) support JSON serialization of cache data (instead of pickle). These might be convert a `deque` to `list` (or throw a `TypeError` on insertion).